docs/API.md

```diff
@@ -344,6 +347,28 @@ Network state and email for a user attempting to authenticate into the app. Exte
 -   **email**: _`string | null`_
     -   The email with which a user is attempting to authenticate into the app.
 
+## LoginErrorDisplayConfig
+
+Custom configuration for the error message display on the login screen.
+
+### Type Declaration
+
+-   **mode** (optional): _`'dialog' | 'message-box' | 'both' | 'none'`_
+    -   Sets whether to show login error messages via a pop-up dialog, an inline message box, both, or neither.
+    -   Default: 'dialog'
+-   **dismissible** (optional): _`boolean`_
```

Comment:
I just went down the rabbit hole on this...I feel like I always spell this with an A but that is apparently not correct.

Reply:
If I didn't have a spellchecker plugin in VS code It would have been spelled with an A.